src/views/product-tutorials-view/server.ts

```diff
@@ -85,11 +78,16 @@ export async function getProductTutorialsViewProps(
     allProductCollections,
     productSlug
   )
+  /**
+   * Process page data, reformatting as needed.
+   * Includes parsing headings, for use with the page's sidecar
+   */
+  const { pageData, headings } = await processPageData(rawPageData)
```

Comment:
I've moved the `headings` work into the `processPageData` function. I figure this might be a good place to do the ["trim down static props size"](https://app.asana.com/0/0/1202182325935203/f) work as well, in the future.